src/startup.rs

```diff
@@ -9,7 +9,8 @@ use tracing_actix_web::TracingLogger;
 
 #[tracing::instrument(skip_all)]
 pub fn run(listener: TcpListener, settings: Settings) -> Result<Server, std::io::Error> {
-    let symbol_manager = Arc::new(create_symbol_manager(settings));
+    let symbol_manager = create_symbol_manager(settings);
```

Comment:
[`Data`](https://docs.rs/actix-web/latest/actix_web/web/struct.Data.html) already internally uses an `Arc` so we do not need to wrap it here.